lxmlbind/base.py

```diff
@@ -46,7 +47,7 @@ def _init_properties(self, **kwargs):
         Initialize property names and default values.
         """
         for class_ in getmro(self.__class__):
-            for name, member in class_.__dict__.iteritems():
+            for name, member in six.iteritems(class_.__dict__):
```

Comment:
I don't think there's a big enough advantage to using `iteritems` vs just `dict.items` to justify the import. I would just switch to `items`.